File: 3375-minimum-operations-to-make-array-values-equal-to-k/3375-minimum-operations-to-make-array-values-equal-to-k.java
class Solution {
    public int minOperations(int[] nums, int k) {
        Set<Integer> uniqueHeights = new HashSet<>();
        for (int n : nums) {
            if (n < k) {
                return -1;
            }
            if (n > k) {
                uniqueHeights.add(n);
            }
        }
        return uniqueHeights.size();
    }
}

File: 3375-minimum-operations-to-make-array-values-equal-to-k/3375-minimum-operations-to-make-array-values-equal-to-k.py
class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        h = set()

        for i in range(len(nums)):
            if nums[i] < k:
                return -1
            h.add(nums[i])
        return len(h) - 1 if k in h else len(h)


        